app/apollo/resolvers/cluster.js

```diff
@@ -477,6 +480,9 @@ const clusterResolvers = {
         const deletedResourceYamlHist = await models.ResourceYamlHist.updateMany({ org_id }, {$set: { deleted: true }}, { upsert: false });
         logger.info({req_id, user, org_id, deletedResourceYamlHist}, 'ResourceYamlHist soft-deletion complete');
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {name: deletedClusters.registration, uuid: deletedClusters.cluster_id};
```

Comment:
The `name` and `registration` object are not the same thing -- registration _includes_ name as an attribute, but it's not the name itself.  This might be more appropriate:
```suggestion
        context.pluginContext = {name: deletedClusters.registration.name, uuid: deletedClusters.cluster_id, registration: deletedClusters.registration};
```
**However**
- note that this function is `deleteClusters` and the object `deletedClusters`, both with an `s` -- it's not just deleting a single cluster it's deleting all clusters in the org/account.  The response from the mongo `deleteMany` call is not going to be a single object with a `registration` attribute.
- subscriptions and resources and resourceYamlHist are also being deleted and information about them should also be included in the `pluginContext`.

Other updates in this PR need to be reviewed to make sure they dont have the same problems.